doc/oper.xml

```diff
@@ -1572,6 +1572,92 @@ gap> DigraphShortestDistance(D, [1, 3], [3, 5]);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="Dominators">
+<ManSection>
+  <Oper Name="Dominators" Arg="digraph, root"/>
+  <Returns>A list of lists </Returns>
+    <Description>
+    <C>Dominators</C> is a function that takes a <A>digraph</A> and a
+    specific root <A>root</A> and returns the dominators of each vertex with
+    respect to the root. The root must be a verex of the digraph. The algorithm
+    is an implementation of the almost linear time algorithm by
+    Thomas Lengauer and Robert Endre Tarjan <Cite Key="LT79"/>. If there is
+    no path from the root to a specific vertex, the output will contain a
+    hole in the corresponding position. The complexity of this algorithm is 
+    <M>O(mlog n)</M> where <M>m</M> is the number of edges and <M>n</M> is the
```

Comment:
My knowledge in complexity is almost zero right now, but since we visit all the nodes in the digraph but only the edges of the subdigraph induced by the reachable vertices, shouldn't we get that "n is the number of nodes and m is the number of edges in the digraph induced by the vertices reachable from the root vertex"?